en/sql/schema/table_stmt.rst

```diff
@@ -792,7 +792,21 @@ Specifying the **DONT_REUSE_OID** option when creating the table will create a r
 Charset and Collation
 ^^^^^^^^^^^^^^^^^^^^^
 
-The charset and collation of the table can be designated in **CREATE TABLE** statement. Please see :ref:`collation-charset-string` for details.
+The charset and collation of the table and colum can be designated in **CREATE TABLE** statement.
```

Comment:
```suggestion
The charset and collation of the table and column can be designated in the **CREATE TABLE** statement.
```

Reply:
Fix typos
"The charset and collation of the table and column can be designated in the **CREATE  TABLE** statement."